tests/iio_readdev.c

```diff
@@ -355,15 +349,15 @@ int main(int argc, char **argv)
 
 	nb_channels = iio_device_get_channels_count(dev);
 
-	if (argc == arg_index + 2) {
+	if (argc == optind + 2) {
```

Comment:
in that case (https://github.com/analogdevicesinc/libiio/pull/151/commits/88ef7db62e2e3892dd8e1df1307025cce2a7cf49#r170843858) , should it be `argc == optind + 1` ?

Reply:
Yes, thanks.